jenkins/Pipeline.gy

```diff
@@ -19,13 +19,26 @@ pipeline {
         // shorthand for the Pbench Server image path, and set a variable for
         // the Server Pod name, based on the image tag, so that we don't
         // interfere with other pods on this executor.
-        PB_SERVER_IMAGE_TAG="""${sh(
+        PB_IMAGE_TAG="""${sh(
             returnStdout: true,
             script: 'echo ${CHANGE_ID:-${BRANCH_NAME}}'
-            )}""".trim()
-        PB_SERVER_IMAGE="${PB_INTERNAL_CONTAINER_REG}/pbench/pbenchinacan-pbenchserver"
-        PB_POD_NAME="pbenchinacan_${PB_SERVER_IMAGE_TAG}"
+            ).trim()}"""
```

Comment:
Was there a functional reason to move the `.trim()` inside? It doesn't appear you're depending on this outside my existing error-path usage to pull the journal...

Reply:
> Was there a functional reason to move the .trim() inside?

Not as such.  It just seemed to belong there (e.g., based on usage examples I'd read).

> It doesn't appear you're depending on this outside my existing error-path usage to pull the journal...

I'm using it in several places in this file, now, one of which is in a direct comparison, so it has to be "trim".